js/rpg_windows/Window_MenuStatus.js

```diff
@@ -43,7 +43,7 @@ Window_MenuStatus.prototype.numVisibleRows = function() {
 
 Window_MenuStatus.prototype.loadImages = function() {
     $gameParty.members().forEach(function(actor) {
-        ImageManager.loadFace(actor.faceName());
+        ImageManager.reserveFace(actor.faceName());
```

Comment:
Is that really OK?

>Window_Status and Window_SkillStatus is used after Window_MenuStatus

As you said, this is just a matter of "scene order".
It's never a problem related to the "scene stack".
So this implementation is beside the point.
On the contrary, your solution threatens the independence of the scene.
The best solution is to add `reserveFace` to only two places.